There seems to be a race condition in these make rules with parallel make

--- origsrc/gtk-doc-1.18/gtk-doc.make	2011-08-25 16:20:22.000000000 -0500
+++ src/gtk-doc-1.18/gtk-doc.make	2011-11-17 17:56:52.058603400 -0600
@@ -278,3 +278,4 @@
 	@$(GTKDOC_REBASE) --online --relative --html-dir=$(distdir)/html
 
 .PHONY : dist-hook-local docs
+.NOTPARALLEL:
